content/docs/capabilities/ppl.mdx

```diff
@@ -76,6 +76,41 @@ If `nor` is used, the user will have access if their email address does not end
   </div>
 </details>
 
+### Multiple Operators in an Action
+
+You _can_ add multiple operators under the same Action block, and it would be valid PPL.
+
+For example, this policy would grant access to anyone with an `example<X>.com` domain in their email address:
```

Comment:
I wonder if this `example<X>.com` is a little confusing? It almost reads like any character could substitute for the \<X\>, which is not the case.

It might be clearer to write out "an email address with the domain example1.com, example2.com, or example3.com".